src/PowerShellEditorServices/Services/DebugAdapter/Debugging/VariableDetails.cs

```diff
@@ -414,4 +428,19 @@ public override string ToString()
             }
         }
     }
+
+    /// <summary>
+    /// A VariableDetails that only returns the raw view properties of the object, rather than its values.
+    /// </summary>
+    internal class VariableDetailsRawView : VariableDetails
```

Comment:
```suggestion
    internal sealed class VariableDetailsRawView : VariableDetails
```

Good to seal internals when you don't need to subclass it. Can always change it later if needed, but sealing leads to better code gen.